snap/info.go

```diff
@@ -186,6 +186,11 @@ func (s *Info) CommonDataHomeDir() string {
 	return filepath.Join(dirs.SnapDataHomeGlob, s.Name(), "common")
 }
 
+// NeedsDevmode retursn whether the snap needs devmode.
+func (s *Info) NeedsDevmode() bool {
```

Comment:
yea, we have tended to call things DevMode and TryMode, maybe that's wrong, either way we should be consistent